Graphics/ShaderTools/CMakeLists.txt

```diff
@@ -143,8 +145,15 @@ if (DXC_SUPPORTED)
     endif()
 
     if (NOT PLATFORM_WIN32)
-        if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR
-            CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+        if (DILIGENT_USE_STATIC_DXC)
+            target_compile_definitions(Diligent-ShaderTools PRIVATE DILIGENT_USE_STATIC_DXC=1)
+            target_link_libraries(Diligent-ShaderTools
+                    PUBLIC
+                    LLVMSupport
+                    dxcompiler
+            )
+        endif()
+        if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU")
             target_link_libraries(Diligent-ShaderTools PRIVATE dl)
```

Comment:
I think that `dl` should not be needed when linking statically - can be in the `else()` branch